src/core/game.js

```diff
@@ -125,11 +130,29 @@ class Game {
     this.keys[e.keyCode] = false
   }
 
+  mouse_move(e) {
+    // Accumulate the deltas for each mousemove event that that fired between
+    // any two ticks. Our +X is left, +Y is up while the browser's +X is to the
+    // right, +Y is down. Inverse the values by subtracting rather than adding.
+    this.mouse_delta.x -= e.movementX,
+    this.mouse_delta.y -= e.movementY
+  }
+
   perform_ticks(ticks_qty) {
-      for(var i=0; i < ticks_qty; i++) {
-        this.last_tick = this.last_tick + this.tick_length;
-        this.update(this.last_tick);
-      }
+    // Mouse delta is measured since the last time this.tick was run.  If there
+    // is more than one tick to perform we need to scale the delta down to
+    // maintain consistent movement.
+    this.mouse_delta.x /= ticks_qty;
+    this.mouse_delta.y /= ticks_qty;
```

Comment:
Is this the right thing to do? My goal was to avoid situations when multiple ticks receive the same `mouse_delta` if we're skipping frames.